jsonschema/tests/test_validators.py

```diff
@@ -191,6 +191,26 @@ def test_invalid_format_default_message(self):
         self.assertIn(repr("thing"), message)
         self.assertIn("is not a", message)
 
+    def test_additionalProperties_false_patternProperties(self):
+        schema = {u"type": u"object",
+                  u"additionalProperties": False,
+                  u"patternProperties": {
+                      u"$abc^": {u"type": u"string"},
+                      u"$def^": {u"type": u"string"}
+                  }}
+        message = self.message_for({u"zebra": 123}, schema,
+                                   cls=Draft4Validator)
+        self.assertEqual(
+            message,
+            "{} does not match any of the regexs: $abc^, $def^".format(
+                repr(u"zebra")))
```

Comment:
Nitpick: can you change the formatting here to not combine parens, e.g.

```
self.assertEqual(
    message,
    "whawefiuhweifuhiweuhfiwuehfiuwheftever".format(
        repr(stuff),
    ),
)
```

so that each one gets a trailing comma?